packages/web-client/app/components/card-pay/safe-chooser-dropdown/index.css

```diff
@@ -19,13 +19,14 @@
 }
 
 .safe-chooser-dropdown .ember-basic-dropdown-content {
+  max-height: 300px;
```

Comment:
Just an arbitrary value that looked good. Questions:
1. Should this be a variable?
2. Should we add this to all dropdowns we have? Otherwise we'll probably have the same issues elsewhere

Reply:
I tend to favour extraction of magic numbers, but there are a lot of them in the CSS already so it wouldn’t be a departure to keep this. I think dropdowns in particular might be ready for some generalised styles but I’m unsure how to accomplish that with the way the styles are per-component at the moment 🤔

Reply:
I think it would be nice to have a way to share generalised styles too! Would be nice to have something that can carry more intent/package things more nicely than variable prefixes.

Reply:
(but yep also don't mind the magic number as is)

Reply:
One other thought on this is that ideal dropdown max height seems dependent on screen size. Perhaps it should be in vh, perhaps with a min-max?

Reply:
What about expressing it like this:

`max-height: calc(var(--field-height)*4.5)`.

This way we just rely on the variable which is expressed in rem which reacts to the screen size.

I think displaying 4 items would be enough. I added 0.5 so that it's clearer there is more to see in the dropdown (and that one should scroll down to reveal it)

![image](https://user-images.githubusercontent.com/273660/133585918-a6cbb532-df7e-41f4-94b7-e37ea2280483.png)